src/PhpSpec/Console/ResultConverter.php

```diff
@@ -33,6 +50,12 @@ public function convert($result)
             case ExampleEvent::PASSED:
             case ExampleEvent::SKIPPED:
                 return 0;
+
+            case ExampleEvent::PENDING:
+                if ($this->io->isIgnorePendingEnabled()) {
+                    return 0;
+                }
+                break;
         }
 
         return 1;
```

Comment:
If we are keep the pending option (or adding others later), we would then need to ensure that each case statement returns something from each conditional control path if we move the default return into the `default` case.  Leaving outside means less control logic within the switch structure.
Having said that, if each error type was going to return it's own error code, this would make more sense?